tests/test_homework.py

```diff
@@ -35,11 +35,14 @@ def course_data(request, client_admin, problem_ids):
     # add course
     add_course(cd.name, cd.teacher)
     # add students and TA
-    client_admin.put(f'/course/{cd.name}',
-                     json={
-                         'TAs': cd.tas,
-                         'studentNicknames': cd.students
-                     })
+    rv = client_admin.put(f'/course/{cd.name}',
+                          json={
+                              'TAs': cd.tas,
+                              'studentNicknames': cd.students
+                          })
+
+    rv = client_admin.get(f'/course/{cd.name}')
```

Comment:
why a get request here?

Reply:
It was used for testing, I forgot to remove it